File: core/objs-core/src/test/java/io/qpointz/poc/objs/core/ObjsCoreTest.java
package io.qpointz.poc.objs.core;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class ObjsCoreTest {

    @Test
    void shouldExposeModuleName() {
        assertThat(ObjsCore.MODULE).isEqualTo("objs-core");
    }
}

File: objs-core/src/test/kotlin/org/poc/objs/core/ObjsCoreTest.kt
package org.poc.objs.core

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class ObjsCoreTest {
    @Test
    fun shouldExposeModuleName() {
        assertThat(ObjsCore.MODULE).isEqualTo("objs-core")
    }
}
